kinto/plugins/accounts/__init__.py

```diff
@@ -29,6 +41,20 @@ def includeme(config):
 
     settings = config.get_settings()
 
+    if settings.get("account_validation", False):
+        config.add_api_capability(
+            "account-validation",
```

Comment:
@leplatrem what do you think about this?

I my opinion, we should restrict to one capability per plugin and instead add other info in the capability of the plugin.

```
accounts": {

    "description": "Manage user accounts.",
    "url": "https://kinto.readthedocs.io/en/latest/api/1.x/accounts.html",
    "id-validation": "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$",
    "reset-password-enable": True

}
```

Reply:
I'll add a boolean for `validation-enabled` (I don't think it's relevant to have a `reset-password-enabled` boolean as it's tied to the account validation).